modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms/smt-schemaregistryreplicator.adoc

```diff
@@ -0,0 +1,16 @@
+= SchemaRegistryReplicator (Redpanda)
+:description: How to set SchemaRegistryReplicator (Redpanda) Single Message Transform for a connector.
+:page-cloud: true
+
+The SchemaRegistryReplicator (Redpanda) is a transform to filter and process Redpanda Schema Registry messages.
```

Comment:
maybe `schemas` instead of `Redpanda Schema Registry messages`. It works for other clusters/SRs as well.